.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We should take a holistic look at all our other tasks and see how they can be better integrated with VSCode in a separate PR. For now, this file should be removed.